src/main/java/uk/gov/justice/digital/domains/model/SourceReference.java

```diff
@@ -1,4 +1,4 @@
-package uk.gov.justice.digital.service.model;
+package uk.gov.justice.digital.domains.model;
```

Comment:
Package naming. For consistency please use the singular e.g. `domain` not `domains`.

Reply:
done